tools/diagnostics/create_service_diagnostics_bundle.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+readonly DCOS_COMMONS_SCRIPT_PATH="tools/diagnostics"
+
+function is_dcos_commons_repository_script() {
+  [[ ${SCRIPT_DIRECTORY} = *${DCOS_COMMONS_SCRIPT_PATH} ]]
```

Comment:
Should one check for a git repo with a valid remote too?

Reply:
I originally had that but removed it since this could be run from an expanded tarball directory too, for example.